### PR TITLE
Google maps SDK for location

### PR DESCRIPTION
**Pull request by nikita206**

### CODE REVIEW

File: StayFit/Signup/SignupViewController.m [hunk 1]
```diff
@@ -136,4 +142,53 @@ - (IBAction)didTapBack:(id)sender {
...
+
+//if the button to add address is clicked then a new view is opened
+- (IBAction)address:(id)sender {
+    [self autocompleteClicked];
+}
+
+- (void)autocompleteClicked {
+  GMSAutocompleteViewController *acController = [[GMSAutocompleteViewController alloc] init];
+  acController.delegate = self;
+  // Specifies the place data types to return.
```
Comment (raghavan-rahul) [inline StayFit/Signup/SignupViewController.m]: dont think this comment is required?

File: StayFit/Tab Bar Controller/Recommended Workout Tab/RecommendedWorkoutViewController.m [hunk 2]
```diff
@@ -132,7 +132,7 @@ -(void) gymBuddy{
            else {
                self.workoutArray = [NSJSONSerialization JSONObjectWithData:data options:NSJSONReadingMutableContainers error:nil];
                self.filteredWorkoutArray = self.workoutArray;
-               //prints out the contents of the API for verification
+              //prints out the contents of the API for verification
```
Comment (raghavan-rahul) [inline StayFit/Tab Bar Controller/Recommended Workout Tab/RecommendedWorkoutViewController.m:135]: same here looks duplicate comments

**Review by raghavan-rahul — APPROVED** [review]
Please remove the additional comments. After that we can merge the PR

File: StayFit/Tab Bar Controller/Recommended Workout Tab/RecommendedWorkoutViewController.m (hunk 2 above)
Comment (nikita206) [reply StayFit/Tab Bar Controller/Recommended Workout Tab/RecommendedWorkoutViewController.m:135]: yes, it's just one comment per if-else statement. Should I remove the same comment from the else blocks and keep it only in the if block?

**Pull request merged**